### ISSUE TITLE
Scanner: Playwright + axe-core integration

### ISSUE THREAD

**Issue by mcull** [mcull/cg_a11y#6]
Render pages with Playwright, inject axe-core, collect violations/incomplete/passes. Capture page title and status.
